### ISSUE TITLE
accuracy is not good, how to improve?

### ISSUE THREAD

**Issue by goodtogood** [likelyzhao/peron_attribute#3]
Hi,there
thanks for your efforts!
after running 105 epochs with pretrained model resnet-50,  Market1501 && Duke
the accuracy detail is as follows

```
Epoch[105] Train-multi-metric-gender_label=0.596175
Epoch[105] Train-multi-metric-hat_label=0.879310
Epoch[105] Train-multi-metric-bag_label=0.797144
Epoch[105] Train-multi-metric-handbag_label=0.923761
Epoch[105] Train-multi-metric-backpack_label=0.552263
Epoch[105] Train-multi-metric-updress_label=0.393588
Epoch[105] Train-multi-metric-downdress_label=0.399515
Epoch[105] Time cost=147.776
Saved checkpoint to "./model/personattr-0106.params"
Epoch[105] Validation-multi-metric-gender_label=0.592518
Epoch[105] Validation-multi-metric-hat_label=0.845533
Epoch[105] Validation-multi-metric-bag_label=0.802775
Epoch[105] Validation-multi-metric-handbag_label=0.922977
Epoch[105] Validation-multi-metric-backpack_label=0.579778
Epoch[105] Validation-multi-metric-updress_label=0.414938
Epoch[105] Validation-multi-metric-downdress_label=0.416170
```
seemed  that it's kinda low, how to improve it ?
thanks !

**Comment by chuanruihu** [comment]
INFO:root:Epoch[1] Train-multi-metric-gender_label=0.517857
INFO:root:Epoch[1] Train-multi-metric-hat_label=0.964286
INFO:root:Epoch[1] Train-multi-metric-bag_label=0.651786
INFO:root:Epoch[1] Train-multi-metric-handbag_label=0.910714
INFO:root:Epoch[1] Train-multi-metric-backpack_label=0.705357
INFO:root:Epoch[1] Train-multi-metric-updress_label=0.330357
INFO:root:Epoch[1] Train-multi-metric-downdress_label=0.410714
INFO:root:Epoch[1] Time cost=230.216
INFO:root:Saved checkpoint to "save/personattr-0002.params"
INFO:root:Epoch[1] Validation-multi-metric-gender_label=0.559299
INFO:root:Epoch[1] Validation-multi-metric-hat_label=0.971341
INFO:root:Epoch[1] Validation-multi-metric-bag_label=0.757317
INFO:root:Epoch[1] Validation-multi-metric-handbag_label=0.905335
INFO:root:Epoch[1] Validation-multi-metric-backpack_label=0.749009
INFO:root:Epoch[1] Validation-multi-metric-updress_label=0.346265
INFO:root:Epoch[1] Validation-multi-metric-downdress_label=0.39451
 

I find the metirc loss is not decrease ,,  who can help me ?

**Comment by goodtogood** [comment]
I got the same problem, after running 190 epochs, 
the validation accuracy roughly looks like unchanged.
![val-acc](https://user-images.githubusercontent.com/10088733/45399850-53077e00-b67c-11e8-9423-620a8f3300ba.jpg)

**Comment by chuanruihu** [comment]
yes, this code may have somewhere  wrong. Hope the writer can solve it. In addition, the prediction folder has http://otr41gcz3.bkt.clouddn.com/personattr.tar  this model  may not to use. Can you try it?

**Comment by goodtogood** [comment]
@chuanruihu  
currently, It seems the code doesn't work well,
so I gave it up .
